test/integration/data_entry_test.rb

```diff
@@ -104,7 +104,7 @@ class DataEntryTest < ActionDispatch::IntegrationTest
         create :patient, primary_phone: '111-111-1111'
 
         select 'DC', from: 'patient_line'
-        fill_in 'Initial call date', with: 2.days.ago.strftime('%Y-%m-%d')
+        fill_in 'Initial call date', with: 2.days.ago.strftime('%m-%d-%Y')
```

Comment:
Might need to swap this to `display_date`. Can probably swap it in the other two changes to the asserts above, too, if we're lucky!